src/mem/cache/prefetch/spatio_temporal_memory_streaming.cc

```diff
@@ -90,15 +92,12 @@ STeMS::checkForActiveGenerationsEnd(const CacheAccessor &cache)
             }
             if (generation_ended) {
                 // PST is indexed using the PC (secure bit is unused)
-                ActiveGenerationTableEntry *pst_entry =
-                    patternSequenceTable.findEntry(pst_addr,
-                                                   false /*unused*/);
+                ActiveGenerationTableEntry *pst_entry = patternSequenceTable.findEntry(pst_addr);
```

Comment:
This line is currently going beyond the character limit

Reply:
Fixed.